packages/react/src/Spotlight/Spotlight.tsx

```diff
@@ -6,9 +6,10 @@
 import clsx from 'clsx'
 import { forwardRef, HTMLAttributes, PropsWithChildren } from 'react'
 
-export const spotlightColors = ['azure', 'blue', 'green', 'lime', 'magenta', 'orange', 'purple', 'yellow'] as const
+const defaultColor = 'purple'
```

Comment:
If this is an unchanging value (as in, a true constant), the convention is to write it in all caps.

Reply:
We decided to postpone this possible change.